create_primers.py

```diff
@@ -240,32 +293,25 @@ def main():
         raise IOError(f"Cannot find mutatcodon_frequency_csvions_csv {codon_frequency_csv}")
 
     # Design forward mutation primers
-    mutforprimers = CreateMutForOligosVarLength(sequence, mutations_csv, codon_frequency_csv, primerlength, primerprefix, args['maxprimertm'], args['minprimertm'], args['maxlength'], args['minlength'])
-    print("Designed %d mutation forward primers." % len(mutforprimers))
+    mutforprimers = CreateMutForOligosVarLength(sequence, mutations_csv,
+        codon_frequency_csv, primerlength, primerprefix, args['maxprimertm'],
+        args['minprimertm'], args['maxlength'], args['minlength'])
+    print(f"Designed {len(mutforprimers)} mutation forward primers.")
 
     # Design reverse mutation primers
-    mutrevprimers = [(name.replace('for', 'rev'), ReverseComplement(seq)) for (name, seq) in mutforprimers]
-    print("Designed %d mutation reverse primers." % len(mutrevprimers))
+    mutrevprimers = [(name.replace('for', 'rev'), ReverseComplement(seq))
+                      for (name, seq) in mutforprimers]
+    print(f"Designed {len(mutrevprimers)} mutation reverse primers.")
 
     # Print out all of the primers
     primers = mutforprimers + mutrevprimers
-    print("This gives a total of %d primers." % len(primers))
+    print(f"This gives a total of {len(primers)} primers.")
 
-    print("\nNow writing these primers to %s" % outfile)
-    iplate = 1
+    print(f"\nNow writing these primers to {outfile}")
     f = open(outfile, 'w')
     for primers in [mutforprimers, mutrevprimers]:
-        f.write("\r\nPlate %d\r\n" % iplate)
-        n_in_plate = 0
         for (name, primer) in primers:
-            f.write("%s, %s\r\n" % (name, primer))
```

Comment:
Do you need the plate functionality? Or maybe it should be an argument for "order in plate" or not? I'm not sure how you order these things and if just a long list is sufficient, but I assume Adam had this functionality for a reason? But that may also be out-of-date.

Reply:
The in-plate ordering functionality was for ordering from IDT in plates and then hand mixing the primers. It seems the best options now are to order oligo pools pre-pooled, which we just need a list of the names and primers for.

Reply:
Oh yeah, that sounds way better! Sweet.